deps/inspector_protocol/pdl.py

```
@@ -86,6 +86,18 @@ def parse(data, file_name, map_binary_to_string=False):
       protocol['domains'].append(domain)
       continue
 
+    match = re.compile(
+        r'^include (.*)').match(line)
+    if match:
+      included_filename = match.group(1)
+      if path.isabs(included_filename):
```

Pylint (4.0.4)

[error] Undefined variable 'path'

(E0602)